components/developer/maven/maven.p5m

```diff
@@ -19,47 +20,43 @@ set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
-license LICENSE license='Apache 2.0'
-
-link path=usr/bin/mvn target=../share/maven/bin/mvn
```

Comment:
Preserve usr/bin/mvn link

Reply:
sorry...I always miss something. should make myself a checklist...:/